gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Sep 24 17:31:03 MDT 2012
+#Mon Jul 06 16:25:58 MDT 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=http:\//services.gradle.org/distributions/gradle-2.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
```

Comment:
Yeah, I guess they just decided to go secure.  This is just what was generated by the 2.3 ./gradlew wrapper command